src/pages/api/profile/user-profile/index.ts

```diff
@@ -0,0 +1,116 @@
+import { getServerSideCookies, Endpoint, HttpStatus } from "@fwl/web";
+import {
+  loggingMiddleware,
+  wrapMiddlewares,
+  tracingMiddleware,
+  errorMiddleware,
+  recoveryMiddleware,
+  rateLimitingMiddleware,
+} from "@fwl/web/dist/middlewares";
+import { NextApiRequest, NextApiResponse } from "next";
+
+import { Handler } from "@src/@types/handler";
+import { UserCookie } from "@src/@types/user-cookie";
+import { configVariables } from "@src/configs/config-variables";
+import { logger } from "@src/configs/logger";
+import { initProfileClient } from "@src/configs/profile-client";
+import getTracer from "@src/configs/tracer";
+import { ERRORS_DATA, webErrorFactory } from "@src/errors/web-errors";
+import { authMiddleware } from "@src/middlewares/auth-middleware";
+import { sentryMiddleware } from "@src/middlewares/sentry-middleware";
+import { getProfileAccessToken } from "@src/utils/get-profile-access-token";
+
+const patchHandler: Handler = async (request, response) => {
+  const webErrors = {
+    invalidProfileToken: ERRORS_DATA.INVALID_PROFILE_TOKEN,
+    invalidScopes: ERRORS_DATA.INVALID_SCOPES,
+    userProfileNotFound: ERRORS_DATA.USER_PROFILE_NOT_FOUND,
+    invalidUserProfilePayload: ERRORS_DATA.INVALID_USER_PROFILE_PAYLOAD,
+    unreachable: ERRORS_DATA.UNREACHABLE,
+  };
+
+  return getTracer().span(
+    "/api/profile/user-profile patchHandler",
```

Comment:
```suggestion
    "PATCH /api/profile/user-profile handler",
```
Maybe more inline with the REST version of writing it?
Wdyt?